benchmarks/user/benchmark.py

```diff
@@ -226,6 +262,10 @@ def bench(problem, **kwargs):
                    'ceil was obtained (ideal peak, linpack, ...)')
 @click.option('--point-runtime', is_flag=True, default=True,
               help='Annotate points with runtime values')
+@click.option('--section', default=None,
+              help='Code section for which the roofline is plotted')
```

Comment:
Is there a way to tell people what this neame look like, like "should be `section_0,1,...` as I am not sure people will know what the names are out of the box

Reply:
I think if they use this option, they are expected to look at the generated C code?

Reply:
I don't know about that, think would still prefer a "naming convention is ...."